trollflow2/tests/test_launcher.py

```diff
@@ -887,5 +887,69 @@ def test_generate_messages():
             assert msg.type in VALID_MESSAGE_TYPES
 
 
+def test_sigterm_generate_messages():
+    """Test that sending sigterm to Trollflow2 stops it."""
+    import os
+    import signal
+    from multiprocessing import Process
+
+    from trollflow2.launcher import generate_messages
+
+    connection_parameters = {"nameserver": False, "addresses": "localhost:40000", "topic": "/test"}
```

Comment:
I've had problems to run tests with connections to localhost on some machines in the past. Would it be possible to switch this with an ipc connection instead?
eg:
```suggestion
def test_sigterm_generate_messages(tmp_path):
    """Test that sending sigterm to Trollflow2 stops it."""
    import os
    import signal
    from multiprocessing import Process

    from trollflow2.launcher import generate_messages
    ipc_path = os.fspath(tmp_path / "my_pipe")
    connection_parameters = {"nameserver": False, "addresses": f"ipc://{ipc_path}", "topic": "/test"}
```

Reply:
The connection is never used, it's basically just saying "don't bother connecting to a nameserver that isn't there". I don't mind changing to your suggestion but I don't understand what it does.

Reply:
ipc is inter-process communication and basically just uses a pipe file on linux. Ie, doesn't need a network to work. It's supported by pyzmq and thus posttroll, I use it in other tests. It's treated as just another protocol.